nio/schemas.py

```diff
@@ -1213,6 +1213,22 @@ class Schemas(object):
         ],
     }
 
+    tags = {
+        "type": "object",
+        "properties": {
+            "type": {"type": "string"},
+            "content": {
```

Comment:
The content should be a required property if you're going to use pop since it'll result in a KeyError if it's missing.